tensorflow_data_validation/utils/slicing_util.py

```diff
@@ -155,41 +153,41 @@ def feature_value_slicer(
                             "the provided slice values are not valid integers."
                         ) from e
 
-                        flattened, value_parent_indices = array_util.flatten_nested(
-                            feature_array, True
-                        )
-                        non_missing_values = np.asarray(flattened)
-                        # Create dataframe with feature value and parent index.
-                        df = pd.DataFrame(
-                            {
-                                feature_name: non_missing_values,
-                                _PARENT_INDEX_COLUMN: value_parent_indices,
-                            }
-                        )
-                        df = df.drop_duplicates()
-                        # Filter based on slice values
-                        if values is not None:
-                            df = df.loc[df[feature_name].isin(values)]
-                        per_feature_parent_indices.append(df)
-                # If there are no features to slice on, yield no output.
-                # TODO(b/200081813): Produce output with an appropriate placeholder key.
-                if not per_feature_parent_indices:
-                    return
-                # Join dataframes based on parent indices.
-                # Note that we want the parent indices per slice key to be sorted in the
-                # merged dataframe. The individual dataframes have the parent indices in
-                # sorted order. We use "inner" join type to preserve the order of the left
-                # keys (also note that same parent index rows would be consecutive). Hence
-                # we expect the merged dataframe to have sorted parent indices per
-                # slice key.
-                merged_df = functools.reduce(
-                    lambda base, update: base.merge(
-                        update,
-                        how="inner",  # pylint: disable=g-long-lambda
-                        on=_PARENT_INDEX_COLUMN,
-                    ),
-                    per_feature_parent_indices,
-                )
+            flattened, value_parent_indices = array_util.flatten_nested(
```

Comment:
@andrewfulton9 Is this intended here, or did I mess this up when I merged `master`?